presto-main/src/test/java/com/facebook/presto/operator/TestClassificationMissRateAggregation.java

```diff
@@ -34,7 +34,7 @@ public TestClassificationMissRateAggregation()
         final ArrayList<Double> expected = new ArrayList<>();
         while (iterator.hasNext()) {
             final BucketResult result = iterator.next();
-            final double missRate = result.remainingFalseWeight / result.totalTrueWeight;
+            final double missRate = (result.totalTrueWeight - result.remainingTrueWeight) / result.totalTrueWeight;
```

Comment:
Could you also define falseNegative before calculating miss rate,  to be consistent with that in `TestClassificationPrecisionAggregation.java`

Reply:
done.